rest_docs/rest-catalog-open-api.yaml

```diff
@@ -0,0 +1,712 @@
+      schema:
+        type: string
+      example: "accounting%00tax"
+
+  ##############################
+  # Application Schema Objects #
+  ##############################
+  schemas:
+
+    ResponseDataObject:
+      type: object
+      description: JSON data payload returned in a successful response body
+      properties:
+        data:
+          type: object
+          description: Wrapper for the response of a successful request
+      example: { "data": { ... } }
+
+    ResponseErrorObject:
+      type: object
+      description: JSON error payload returned in a response with further details on the error
+      required:
+        - message
+        - type
+        - code
+      properties:
+        message:
+          type: string
+          description: Human-readable error message
+        type:
+          type: string
+          description: Internal type definition of the error
+          example: NoSuchNamespaceException
+        code:
+          type: integer
+          minimum: 400
+          maximum: 600
+          description: HTTP response code
+          example: 404
+
+    CatalogConfiguration:
+      type: object
+      description: Server-provided configuration for the catalog.
+      required:
+        - properties
+        - defaults
+      properties:
+        overrrides:
+          type: object
+          nullable: false
+          description:
+            Properties that should be used to override client configuration; applied after defaults and client configuration.
+        defaults:
+          type: object
+          nullable: false
+          description:
+            Properties that should be used as default configuration; applied before client configuration.
+
+    CreateNamespaceRequest:
+      type: object
+      required:
+        - namespace
+      properties:
+        namespace:
+          $ref: '#/components/schemas/Namespace'
+        properties:
+          type: object
+          description: Configured string to string map of properties for the namespace
+          example: '{ "owner": "Hank Bendickson" }'
+          default: '{ }'
+
+    Namespace:
+      description: Reference to one or more levels of a namespace
+      type: array
+      items:
+        type: string
+      example: [ "accounting", "tax" ]
+
+    RenameTableRequest:
+      type: object
+      properties:
+        sourceTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+        destinationTableIdentifier:
+          $ref: '#/components/schemas/TableIdentifier'
+
+    TableIdentifier:
+      type: object
+      required:
+        - namespace
+        - name
+      properties:
+        namespace:
+          type: array
+          description: Individual levels of the namespace
+          items:
+            type: string
+          nullable: false
+        name:
+          type: string
+          nullable: false
+
+    UpdatePropertiesRequest:
+      type: object
+      properties:
+        removables:
```

Comment:
Let's go with `removals` instead. I just wouldn't use the `-able` suffix here.

Reply:
Removed.